src/scope.ts

```diff
@@ -138,24 +138,36 @@ export class Scope<T extends typeof JSORMBase = typeof JSORMBase> {
     return copy
   }
 
-  select(clause: FieldScope) {
+  select(clause: FieldScope | string[]) {
```

Comment:
would put the string array into the `FieldScope` type, otherwise it will be more annoying to integrate with in places like [the model class](https://github.com/jsonapi-suite/jsorm/blob/1.x-dev/src/model.ts#L721-L727). You'll probably have to change the [internal type](https://github.com/jsonapi-suite/jsorm/pull/81/files#diff-24c72bb2f8cce2d92bbc5c26b978523aL45)

Reply:
The problem is that the internal type (`this._fields`) is actually always an object. Only the argument can be an array of strings, and it is converted to an object within the method. We can create a separate `FieldScopeArg` type but I'm not sure that's better.

Reply:
Updated the model args with `| string[]` but will create a `FieldScopeArg` if you think that's better.